src/main/java/ch/njol/skript/expressions/ExprOption.java

```diff
@@ -0,0 +1,99 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.util.Kleenean;
+import org.apache.commons.lang.SerializationUtils;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Name("Script Options")
+@Description("It returns one or more options from any script.")
+@Examples({
+	"options:",
+	"\ttest: Hello World",
+	"",
+	"command /test:",
+	"\ttrigger:",
+	"\t\tsend \"The option is %skript option \"test\"%\" in \"test.sk\""
+})
+@Since("INSERT VERSION")
+public class ExprOption extends SimpleExpression<String> {
+
+	static {
+		Skript.registerExpression(ExprOption.class, String.class, ExpressionType.COMBINED, "[the] [s(c|k)ript] option[s] %strings% (from|in) [s(k|c)ript] %string%");
+	}
+
+	public static void updateOptions(String scriptName, Map<String, String> options){
+		SKRIPT_OPTIONS.put(scriptName, (Map<String, String>) SerializationUtils.clone(options));
```

Comment:
There is an issue where SerializationUtils is erroring because now Map is an interface and not a Serializable object like HashMap is. Rather than using the SerializationUtils class, we can just copy the map using Java's Map.copyOf method.
```suggestion
		SKRIPT_OPTIONS.put(scriptName, Map.copyOf(options));
```